### ISSUE TITLE
Suggestions Required 👍

### ISSUE THREAD

**Issue by asp2809** [asp2809/awesome-interview-questions#9]
Any Suggestions regarding how to make this repository better whether it's in terms of `Structure`, `QA Format` or anything else is welcome.

**Comment by rhythm-3099** [comment]
This could also include standard data structures and algorithms questions typically asked in the software engineering interviews.

**Comment by asp2809** [comment]
Yes @rhythm-3099, it can, definitely, you just have to create a PR with a new folder named the topic which you want to add with a README.md file in it containing questions with their answers.
Also, if you want to share the Pseudo Code/Code, you can add a link within your answer containing the same.
